.gitlab-ci.yml

```diff
@@ -69,3 +96,70 @@ test:acceptancetests:
 
   script:
     - ./acceptance-tests/run.sh
+
+changelog:
+  image: ${CI_DEPENDENCY_PROXY_DIRECT_GROUP_IMAGE_PREFIX}/node:20
+  stage: changelog
+  tags:
+    - hetzner-amd-beefy
+  rules:
+    - if: $CI_COMMIT_BRANCH =~ "/^\d+\.\d+\.x$/"
+    - if: $CI_COMMIT_BRANCH == $DEFAULT_BRANCH
+  script:
+    - npm install -g release-please
+    - release-please release-pr
+        --token=${GITHUB_BOT_TOKEN_REPO_FULL}
+        --repo-url=${GITHUB_REPO_URL}
+        --target-branch=${CI_COMMIT_REF_NAME}
```

Comment:
as a more general remark: IMO we could add the changelog customization here:
- generate changelog
- get the remote branch
- replace the release please changelog with the generated one
- push again

Reply:
Thanks for the hint; I think I implemented the suggestions, please review again.